### PR TITLE
[f] East - Add rightLabel, searchable, searchable, sorting props into SelectionType

### PR DESCRIPTION
**Pull request by VinhVu0412**
## Description
- [(2) Toast: Add new dynamic multi-selects to Toast](https://trello.com/c/RLvp59Jg/2789-2-toast-add-new-dynamic-multi-selects-to-toast)

## Related raydiant-element PRs
- [[f] East - Add searching and bulk selection into MultiSelect and SelectionInput](https://github.com/mirainc/raydiant-elements/pull/403)
- [[f] East - Add rightLabel to MultiSelect](https://github.com/mirainc/raydiant-elements/pull/406)
- [[f] East - Add row-selected state to MultiSelect](https://github.com/mirainc/raydiant-elements/pull/407 )

### CODE REVIEW

File: packages/raydiant-kit/src/prop-types/SelectionType.ts [hunk 1]
```diff
@@ -3,14 +3,29 @@ import BaseType, { PropType } from './BaseType';
...
   value: string;
+  rightLabel?: string;
   thumbnailUrl?: string;
 }
 
+type SortKey = 'default' | 'label' | 'rightLabel';
+
+interface SortConf {
+  name: string,
+  isNumeric?: boolean,
```
Comment (jschr) [inline packages/raydiant-kit/src/prop-types/SelectionType.ts:14]: This should really be `type: number | string | boolean`. Why did we only make this work for numbers?

File: packages/raydiant-kit/src/prop-types/SelectionType.ts [hunk 2]
```diff
@@ -3,14 +3,29 @@ import BaseType, { PropType } from './BaseType';
...
+}
+
+type Sorting = {
+  [K in SortKey]?: SortConf;
+}
```
Comment (jschr) [inline packages/raydiant-kit/src/prop-types/SelectionType.ts:19]: Does this need to be an object? This feels like it should be an array so we can guarantee order is preserved.

Change to: `sortable?: SortOptions[]`

Comment (jschr) [reply packages/raydiant-kit/src/prop-types/SelectionType.ts:19]: See https://github.com/mirainc/raydiant-elements/pull/406#discussion_r572102424

Comment (VinhVu0412) [reply packages/raydiant-kit/src/prop-types/SelectionType.ts:15]: Thank @jschr. I have just replaced `sorting` with `sortable`.

**Review by sebko — APPROVED** [review]

**Pull request merged**